tools/StaticAnalysis/Exceptions/Az.CosmosDB/SignatureIssues.csv

```diff
@@ -35,4 +35,5 @@
 "Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll","Microsoft.Azure.Commands.CosmosDB.RemoveAzCosmosDBSqlUserDefinedFunction","Remove-AzCosmosDBSqlUserDefinedFunction","1","8510","Cmdlet 'Remove-AzCosmosDBSqlUserDefinedFunction' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
 "Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll","Microsoft.Azure.Commands.CosmosDB.RemoveAzCosmosDBTable","Remove-AzCosmosDBTable","1","8510","Cmdlet 'Remove-AzCosmosDBTable' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
 "Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll","Microsoft.Azure.Commands.CosmosDB.NewAzCosmosDBAccount","New-AzCosmosDBAccount","1","8410","Parameter NetworkAclBypass of cmdlet New-AzCosmosDBAccount does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
-"Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll","Microsoft.Azure.Commands.CosmosDB.UpdateAzCosmosDBAccount","Update-AzCosmosDBAccount","1","8410","Parameter NetworkAclBypass of cmdlet Update-AzCosmosDBAccount does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
\ No newline at end of file
+"Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll","Microsoft.Azure.Commands.CosmosDB.UpdateAzCosmosDBAccount","Update-AzCosmosDBAccount","1","8410","Parameter NetworkAclBypass of cmdlet Update-AzCosmosDBAccount does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll","Microsoft.Azure.Commands.CosmosDB.NewAzCosmosDBDatabaseToRestore","New-AzCosmosDBDatabaseToRestore","1","8100","New-AzCosmosDBDatabaseToRestore Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```

Comment:
Please add set shouldprocess=true for NewAzCosmosDBDatabaseToRestore

Reply:
Removed the change. Actually, this cmd (New-AzCosmosDBDatabaseToRestore) doesn't do any processing. It just creates the list of databases/collections to restore which gets used by restore cmdlet and have shouldprocess=true

Reply:
I had to keep this line for fixing static analysis error. But for this api, we don't need shouldprocess=true for this api as this api isn't an action api. Its just a helper api for RestoreAzCosmosDBDatabase api.